src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/configuration/TemplateIdentifier.java

```diff
@@ -9,7 +9,11 @@
 public enum TemplateIdentifier {
     JSON("application/json", "geojson-template.json"),
     GEOJSON("application/geo+json", "geojson-template.json"),
-    JSONLD("application/ld+json", "json-ld-template.json");
+    JSONLD("application/ld+json", "json-ld-template.json"),
+    GML32("application/gml+xml;version=3.2", "gml32-template.xml"),
+    GML31("gml3", "gml31-template.xml"),
+    GML2("GML2text/xml;subtype=gml/2.1.2", "gml2-template.xml"),
+    XHTML("text/html", "xhtml-template.xhtml");
```

Comment:
I see this mentioned in a couple of places, but does not seem like a response in XHTML is actually available. Which is an interesting bit I guess, while json and GML are existing responses that can be intercepted and replaced with the templated output, XHTML is brand new, was not there before.

Reply:
it is a leftover, I removed it.